### ISSUE TITLE
Save on Records continues to Spin but Record is actually Created

### ISSUE THREAD

**Issue by annikaLiving** [bcgov/ckan-ui#309]
When completing a record's required fields and then hitting save the page continues to spin but if you click out of this and search for the record, it has been created.

**Comment by SteveChapmanBCDX** [comment]
This issue appears to be intermittent.

I ran two tests.  The second test reproduced the bug.

**Test #1:**

I could not reproduce this issue.  I followed @annikaLiving's steps to reproduce the issue.  On clicking "Save", I received a notification that my record was successfully created.

**Test #2:**

I followed the steps to reproduce a second time.  I was able to reproduce the issue.

I did not see any useful information in either the console logs or the network logs.
